src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderFlowValidator.java

```diff
@@ -136,6 +140,9 @@ private Future<Void> validateLocationRestrictions(CompositePoLine poLine, List<F
     }
 
     Set<String> validLocations = poLine.getLocations().stream().map(Location::getLocationId).collect(Collectors.toSet());
+    List<String> holdingIds = poLine.getLocations().stream().map(Location::getHoldingId).collect(Collectors.toList());
+    List<String> permanentLocationIdFromHoldings = getPermanentLocationIdFromHoldings(holdingIds, requestContext);
```

Comment:
we should not do call to mod-inventory if locations are not restricted

Reply:
Avoid calling mod-inventory in case of there is no restrictions , Thanks